straxen/plugins/local_minimum_info.py

```diff
@@ -0,0 +1,196 @@
+
+        """
+        max_gos = np.nan
+        num_loc_maxes = 0
+        valley_gap = np.nan
+        valley_height_ratio = np.nan
+
+        if event['s2_area'] > 0:
+            p = peaks[event['s2_index']]
+
+            smoothing_number = p['width'][9] / p['dt']
+            smoothing_number = np.ceil(smoothing_number / self.config['divide_90p_width'])
+            smoothed_peak = power_smooth(p['data'][:p['length']],
+                                         int(smoothing_number),
+                                         self.config['smoothing_power'])
+
+            # Set data below percentage threshold on both side to zeros
+            left, right = bounds_above_percentage_height(smoothed_peak,
+                                                         self.config['percentage_threshold'])
+
+            # Maximum GOS calculation for data above percentage
+            max_gos = np.max(natural_breaks_gof(p['data'][left: right],
+                                                p['dt']))
+
+            # Local minimum based information
+            maxes, mins = identify_local_extrema(smoothed_peak)
+
+            maxes = maxes[np.logical_and(maxes >= left, maxes < right)]
+            mins = mins[np.logical_and(mins >= left, mins < right)]
+
+            num_loc_maxes = len(maxes)
+
+            valley_gap, valley = full_gap_percent_valley(smoothed_peak,
+                                                         maxes,
+                                                         mins,
+                                                         self.config['percent_valley_height'],
+                                                         p['dt'])
+
+            valley_height_ratio = valley / np.max(smoothed_peak)
+
+        return {'time': event['time'],
+                'endtime': event['endtime'],
+                's2_max_gos': max_gos,
+                's2_num_loc_max': num_loc_maxes,
+                's2_valley_gap': valley_gap,
+                's2_valley_height_ratio': valley_height_ratio}
+
+
+def full_gap_percent_valley(smoothp, max_loc, min_loc, pv, dt):
+    """
+    Full gap at percent valley. The width of the valley at "pv" of the valley height
+    Only do this for peaks which have number of maxes-number of mins = 1, since otherwise
+    this local minimum finding doesn't make sense. Furthermore, it gets rid of those peaks
+    which start at some high value likely because they are the tails of another peak
+    or something.
+    :param smoothp: The smoothed peak
+    :param max_loc: Location of every local maximum of the peak
+    :param min_loc: Location of every local minimum of the peak
+    :param pv: "Percent value", the percent of the valley height for which to calculate the gap
+    :param dt: The time of one sample in ns
+    :return: The gap in ns, the depth in PE
+    """
+    n_gap = len(min_loc)
+    p_length = len(smoothp)
+    if (len(max_loc) - n_gap) == 1 and len(min_loc):

```

Comment:
You could invert this statement and directly return 0,0. In this way you can remove the additional else and indentation.